InfluxDb.cpp

```diff
@@ -81,10 +81,26 @@ void Influxdb::setVersion(uint16_t version){
   begin();
 }
 
+/**
+ * Set servers finger print for HTTPS v2 Influx proto
+ * @param version accepts 1 for version 1.x or 2 for version 2.x
+ */
+void Influxdb::setFingerPrint(char *fingerPrint){
+  _fingerPrint = fingerPrint;
+  begin();
+}
+
 void Influxdb::begin() {
   // TODO: recreate HttpClient on db change?
   if(_db_v == 2){
-    http.begin(_host, _port, "/api/v2/write?org=" + _org + "&bucket=" + _bucket);
+    if (_port == 443) {
+        if (_fingerPrint)
+          client.setFingerprint(_fingerPrint);
+        http.begin(client, _host, _port, "/api/v2/write?org=" + _org + "&bucket=" + _bucket,
```

Comment:
Maybe put that on one line
```suggestion
        http.begin(client, _host, _port, "/api/v2/write?org=" + _org + "&bucket=" + _bucket, true);
```

Reply:
sure